docs/api/modules/testing.md

```diff
@@ -2,8 +2,63 @@
 
 ## `createTestContext`
 
-todo
+Setup a test context providing utilities to query against your GraphQL API.
+
+This context can be augmented by plugins.
+
+#### Signature
+
+```ts
+function createTestContext(opts?: CreateTestContextOptions): Promise<TestContext>
+```
+
+#### Examples
+
+With Jest
+
+```ts
+import { } from 'nexus/testing'
+
+let ctx: TestContext
+
+beforeAll(async () => {
+  ctx = await createTestContext()
+  await ctx.server.start()
+})
+
+afterAll(async () => {
+  await ctx.server.stop()
+})
+
+test('hello', async () => {
+  const result = await ctx.query(`{ hello }`)
+
+  expect(result).toMatchInlineSnapshot()
+})
+```
+
+## `I` CreateTestContextOptions
+
+```ts
+export interface CreateTestContextOptions {
+  /**
+   * A path to the entrypoint of your app. Only necessary if the entrypoint falls outside of Nexus convention.
+   * You should typically use this if you're using `nexus dev --entrypoint` or `nexus build --entrypoint`.
+   */
```

Comment:
Looks like its soon time to start talking about meta config that cannot exist in code. Because the direction now is we'll end up in situations where npm scripts encode nexus commands using `-e` flag, then syncing that with test code here, users forgetting, etc. and its so low-level that we cannot really provide great feedback for users about this.

For example being able to write:

```json
"nexus": {
  "core": {
    "entrypoint": "..."
  }
}
```

which will be honoured everywhere will become a useful feature is my point.